info.json

```diff
@@ -0,0 +1,271 @@
+        },
+        {
+          "title": "Added After",
+          "name": "added_after",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "datetime",
+          "tooltip": "Filter the result set to only include the items added after the specified time.",
+          "description": "Filter the result set to only include the items added after the specified time."
+        },
+        {
+          "title": "Save Response in File",
+          "name": "file_response",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "checkbox",
+          "onchange": {
+            "true": [
+              {
+                "title": "Filename",
+                "name": "filename",
+                "type": "text",
+                "visible": true,
+                "editable": true,
+                "required": false,
+                "placeholder": "Eg. demo.json",
+                "tooltip": "file will create under /tmp directory"
+              }
+            ]
+          },
+          "tooltip": "If check then save the response in File else in JSON",
+          "description": "Specify the response field, If check then save the response in File else in JSON"
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "get_output_schema",
+      "annotation": "get_output_schema",
+      "description": "Gives the output schema based on the inputs",
+      "title": "Get Output Schema",
+      "enabled": true,
+      "visible": false,
+      "parameters": [],
+      "output_schema": {}
+    },
+    {
+      "operation": "get_objects_by_object_id",
+      "title": "Get Object By Object ID",
+      "description": "Returns general information for the specific object in one collection.",
+      "category": "investigation",
+      "annotation": "get_objects_by_object_id",
+      "output_schema": {
+        "id": "",
+        "objects": [
+          {
+            "created": "",
+            "description": "",
+            "id": "",
+            "labels": [
+            ],
+            "modified": "",
+            "name": "",
+            "object_marking_refs": [
+            ],
+            "pattern": "",
+            "type": "",
+            "valid_from": ""
+          },
+          {
+            "created": "",
+            "definition": {
+              "tlp": ""
+            },
+            "definition_type": "",
+            "id": "",
+            "type": ""
+          }
+        ],
+        "spec_version": "",
+        "type": ""
+      },
+      "parameters": [
+        {
+          "title": "collection ID",
+          "name": "collectionID",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Returns general information about the collection, such as id, title, description, can_read, can_write and media_types.",
```

Comment:
does this action return collection info?

Reply:
No,  It retrieves specific object information.
Changed tooltip and description tp "Specify the Collection ID, who's Object information you want to retrieve"